lib/views/teach_word_view.dart

```diff
@@ -22,21 +37,31 @@ class _TeachWordViewState extends State<TeachWordView>
   late StrokeOrderAnimationController _strokeOrderAnimationControllers;
   late TabController _tabController;
   FlutterTts ftts = FlutterTts();
-
-  String svg = "";
+  late Word wordd;
```

Comment:
Is it a typo? Please use a clear variable name to specify the function of word, so that other engineers can understand it. Ex. newWord.